Zend/zend.c

```diff
@@ -1765,3 +1767,31 @@ ZEND_API void zend_map_ptr_extend(size_t last)
 		CG(map_ptr_last) = last;
 	}
 }
+
+int zend_startup_error_notify_callbacks()
+{
+	zend_llist_init(&zend_error_notify_callbacks, sizeof(zend_error_notify_cb), NULL, 1);
+
+	return SUCCESS;
+}
```

Comment:
Is there a reason this isn't void? If it can only succeed I see no point in it returning ``int``, worst case this can be changed at a later date